chapter07/lambda_function.py

```diff
@@ -0,0 +1,15 @@
+
+import boto3
+ec2 = boto3.client('ec2')
+
+def lambda_handler(event, context):
+
+  userName = event['detail']['userIdentity']['arn'].split('/')[1]
+  instanceId = event['detail']['responseElements']['instancesSet']['items'][0]['instanceId']    
+  print("Adding owner tag " + userName + " to instance " + instanceId + ".")
+  try:
```

Comment:
is the try catch needed? or does lambda also print out the stack trace by default?

Reply:
try..catch is not needed. http://docs.aws.amazon.com/lambda/latest/dg/python-exceptions.html